packages/vats/test/localchain.test.js

```diff
@@ -0,0 +1,156 @@
+import { test as anyTest } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+import { reincarnate } from '@agoric/swingset-liveslots/tools/setup-vat-data.js';
+import { E } from '@endo/far';
+import { M, matches } from '@endo/patterns';
+import { makeDurableZone } from '@agoric/zone/durable.js';
+import { AmountMath, AssetKind, makeIssuerKit } from '@agoric/ertp';
+import { buildRootObject } from '../src/vat-bank.js';
+import { prepareLocalChainTools } from '../src/localchain.js';
+
+/** @type {import('ava').TestFn<ReturnType<makeTestContext>>} */
+const test = anyTest;
+
+const makeTestContext = async t => {
+  const { fakeVomKit } = reincarnate({ relaxDurabilityRules: false });
+  const provideBaggage = key => {
+    const root = fakeVomKit.cm.provideBaggage();
+    const zone = makeDurableZone(root);
+    return zone.mapStore(`${key} baggage`);
+  };
+  const baggage = provideBaggage('localchainTest');
+  const bankVat = E(buildRootObject)(null, null, baggage);
+
+  const zone = makeDurableZone(baggage);
+
+  /** @type {undefined | ERef<import('../src/types.js').BridgeHandler>} */
+  let bankHandler;
+
+  /** @type {import('../src/types.js').ScopedBridgeManager} */
+  const fakeBridgeManager = zone.exo('fakeBankBridgeManager', undefined, {
```

Comment:
going down to the bridge layer seems like over-kill.

consider a simpler mock of just the bankManager such as https://github.com/Agoric/agoric-sdk/blob/303a9f20404106fe9c09befbd728ed7a20ccf7e0/packages/inter-protocol/test/psm/psm.test.js#L734

Reply:
oops... I take that back... I see `makeLocalChain` needs a bridge manager.